### PR TITLE
SC: remove uri in requetValueTransferEvent and add the routine to get uri #201

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes

This PR is derived by merged [PR](https://github.com/klaytn/klaytn/pull/201) at master branch.

```
This PR removed uri in requetValueTransferEvent and add the routine to get uri before handling value transfer.

This PR can reduce the gas used of request value transfer transactions.
```

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by KimKyungup** [comment]
@jeonghonah PTAL. This PR was already reviewed for master branch. 
I cherry-picked commit and added a commit for a conflict.

File: node/sc/bridge_test.go [hunk 1]
```diff
@@ -632,7 +632,7 @@ func TestExtendedBridgeAndCallbackERC721(t *testing.T) {
 		assert.Equal(t, bobAcc.From, ev.To)
 
 		// HandleERC20Transfer
-		tx, err = b.HandleERC721Transfer(bridgeAccount, ev.Raw.TxHash, ev.From, ev.To, ev.TokenAddress, ev.ValueOrTokenId, ev.RequestNonce, ev.Raw.BlockNumber, ev.Uri, ev.ExtraData)
+		tx, err = b.HandleERC721Transfer(bridgeAccount, ev.Raw.TxHash, ev.From, ev.To, ev.TokenAddress, ev.ValueOrTokenId, ev.RequestNonce, ev.Raw.BlockNumber, "", ev.ExtraData)
```
Comment (jeonghonah) [inline node/sc/bridge_test.go]: Minor comment:
We can test `TokenURI()` method here.

Comment (KimKyungup) [reply node/sc/bridge_test.go]: Yes. But, I wanted to test with `bridgeInfo.processingPendingRequestEvents()`. :)

Comment (KimKyungup) [reply node/sc/bridge_test.go]: So I added additional TC. :)

**Pull request merged**